srcpkgs/pinephone-utils/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'pinephone-utils'
+pkgname=pinephone-utils
+version=0.1
+revision=1
+archs="aarch64*"
+create_wrksrc=yes
+depends="alsa-ucm-conf"
+short_desc="Scripts/configs for managing pinephone hardware"
+maintainer="John Sullivan <jsullivan@csumb.edu>"
+license="MIT"
+homepage="https://postmarketos.org/"
+
+_commit_pmos=c48037d5ed307d865903ef3aecd302f1fa4417f8
+_pmos_device='device/main/device-pine64-pinephone'
+_pmos_checkout="pmaports-${_commit_pmos}-$(echo ${_pmos_device} | tr '/' '-')/${_pmos_device}"
```

Comment:
instead of the subshell, use `${_pmos_device//\//-}`

Reply:
done :+1: